### ISSUE TITLE
Margin and Padding Broad CSS Rule

### ISSUE THREAD

**Issue by raygig** [stephanrauh/ngx-extended-pdf-viewer#1120]
Currently in viewer-with-images.scss there is some css that is removing all padding and margins from all child elements. I am creating a custom toolbar and this css rule is causing me to apply css overrides to deal with this very broad rule. A simple example is the padding in mat-rasied-buttons. 

Can you please remove this css rule?

```css
ngx-extended-pdf-viewer .html * {
    padding: 0;
    margin: 0;
}
```

**Comment by stephanrauh** [comment]
Consider it done! To my surprise, eliminating this rule changes almost nothing. You can help me by double-checking this and by completing the list of HTML element the rule does influence:

- [ ] Layers menu is affected by eliminating the margins rule
- [ ] ...

**Comment by stephanrauh** [comment]
I've narrowed the scope of the CSS rule slightly. Does this already help you?

**Comment by stephanrauh** [comment]
The feature has landed with version 12.1.0.
